model/yeast-GEM.yml

```diff
@@ -110205,3589 +110205,5808 @@
       - name: "GPI12"
+      - annotation: !!omap
+          - Uniprot ID: "P23797"
     - !!omap
       - id: "YMR289W"
       - name: "ABZ2"
+      - annotation: !!omap
+          - Uniprot ID: "Q03266"
     - !!omap
       - id: "YMR293C"
       - name: "HER2"
+      - annotation: !!omap
+          - Uniprot ID: "Q03557"
     - !!omap
       - id: "YMR296C"
       - name: "LCB1"
+      - annotation: !!omap
+          - Uniprot ID: "P25045"
     - !!omap
       - id: "YMR298W"
       - name: "LIP1"
+      - annotation: !!omap
+          - Uniprot ID: "Q03579"
     - !!omap
       - id: "YMR300C"
       - name: "ADE4"
+      - annotation: !!omap
+          - Uniprot ID: "P04046"
     - !!omap
       - id: "YMR301C"
       - name: "ATM1"
+      - annotation: !!omap
+          - Uniprot ID: "P40416"
     - !!omap
       - id: "YMR303C"
       - name: "ADH2"
+      - annotation: !!omap
+          - Uniprot ID: "P00331"
     - !!omap
       - id: "YMR306W"
       - name: "FKS3"
+      - annotation: !!omap
+          - Uniprot ID: "Q04952"
     - !!omap
       - id: "YMR313C"
       - name: "TGL3"
+      - annotation: !!omap
+          - Uniprot ID: "P40308"
     - !!omap
       - id: "YMR318C"
       - name: "ADH6"
+      - annotation: !!omap
+          - Uniprot ID: "Q04894"
     - !!omap
       - id: "YMR319C"
       - name: "FET4"
+      - annotation: !!omap
+          - Uniprot ID: "P40988"
     - !!omap
       - id: "YMR322C"
       - name: "SNO4"
+      - annotation: !!omap
+          - Uniprot ID: "Q04902"
     - !!omap
       - id: "YMR323W"
       - name: "ERR3"
+      - annotation: !!omap
+          - Uniprot ID: "P42222"
     - !!omap
       - id: "YNL003C"
       - name: "PET8"
+      - annotation: !!omap
+          - Uniprot ID: "P38921"
     - !!omap
       - id: "YNL009W"
       - name: "IDP3"
+      - annotation: !!omap
+          - Uniprot ID: "P53982"
     - !!omap
       - id: "YNL029C"
       - name: "KTR5"
+      - annotation: !!omap
+          - Uniprot ID: "P53966"
     - !!omap
       - id: "YNL036W"
       - name: "NCE103"
+      - annotation: !!omap
+          - Uniprot ID: "P53615"
     - !!omap
       - id: "YNL037C"
       - name: "IDH1"
+      - annotation: !!omap
+          - Uniprot ID: "P28834"
     - !!omap
       - id: "YNL038W"
       - name: "GPI15"
+      - annotation: !!omap
+          - Uniprot ID: "P53961"
     - !!omap
       - id: "YNL045W"
       - name: "LAP2"
+      - annotation: !!omap
+          - Uniprot ID: "Q10740"
     - !!omap
       - id: "YNL048W"
       - name: "ALG11"
+      - annotation: !!omap
+          - Uniprot ID: "P53954"
     - !!omap
       - id: "YNL052W"
       - name: "COX5A"
+      - annotation: !!omap
+          - Uniprot ID: "P00424"
     - !!omap
       - id: "YNL054W"
       - name: "VAC7"
       - annotation: !!omap
-          - kegg.genes: "sce:YNL054W"
-          - ncbigene: "855673"
-          - ncbiprotein: "NP_014344"
-          - refseq: "NM_001182893"
```

Comment:
Not sure wether it is a good idea to remove all other annotations? Maybe an addition of Uniprot ID is better than replacement.

Reply:
I will recheck the codes. Maybe there are some mistake.

Reply:
Problem solved